packages/react-integration/cypress/integration/alert.spec.ts

```diff
@@ -64,19 +33,4 @@ describe('Alert Demo Test', () => {
         cy.wrap(tooltipLink).trigger('mouseleave');
       });
   });
-
-  it('Verify expandable alert', () => {
```

Comment:
I think this is valid to keep.  it verifies the expand works no?

Reply:
Yep! I'll add it back.